src/proxy/http/HttpSM.cc

```diff
@@ -8043,9 +8043,8 @@ HttpSM::set_next_state()
   }
 
   case HttpTransact::SM_ACTION_CACHE_ISSUE_WRITE: {
-    ink_assert((cache_sm.cache_write_vc == nullptr) || t_state.redirect_info.redirect_in_process);
```

Comment:
Am I missing something? This looks like the assertion went off and someone tweaked it so it would stop going off. Issuing a new write while an existing one is in progress is always going to be bad. This bug could've been verified and fixed ages ago if this blaring alarm hadn't been silenced.